pandas/io/json/_json.py

```diff
@@ -1245,6 +1246,9 @@ def _parse(self) -> None:
                 str(k): v
                 for k, v in loads(json, precise_float=self.precise_float).items()
             }
```

Comment:
need tests for this!

Reply:
The test I updated is already testing this. There was a test specific for duplicate columns that was parametrized for orient split, and check the column names. Am I missing something?

Reply:
@jreback do you mind having another look, and letting me know if the added test and my previous comment address your concern. This is ready to be merged pending your review. Thanks!